model/atmosphere/diffusion/tests/diffusion/fixtures.py

```diff
@@ -35,3 +38,33 @@
     stretch_factor,
     top_height_limit_for_maximal_layer_thickness,
 )
+
+
+@pytest.fixture
```

Comment:
this is a small refactoring: instead of using `interpolation_savepoint` directly I use the `interpolation_state` as fixture in the test. this pattern should probably used more, but I only applied it here for now.